src/custom/state/claim/hooks/index.ts

```diff
@@ -346,7 +344,7 @@ export function useUsdcPrice(): string | null {
 /**
  * Generic hook for fetching contract value for the many prices
  */
-function _useVCowPriceForToken(priceFnName: 'wethPrice' | 'gnoPrice' | 'usdcPrice'): string | null {
```

Comment:
make a type for this pls

Reply:
https://github.com/gnosis/cowswap/pull/2205